tests/sentry/api/endpoints/test_admin_project_configs.py

```diff
@@ -0,0 +1,106 @@
+from django.urls import reverse
+from rest_framework import status
+
+from sentry.relay import projectconfig_cache
+from sentry.testutils import APITestCase
+from sentry.testutils.silo import no_silo_test
+
+
+@no_silo_test
+class AdminRelayProjectConfigsEndpointTest(APITestCase):
+    def setUp(self):
+        super().setUp()
+        self.owner = self.create_user(
+            email="example@example.com", is_superuser=False, is_staff=True, is_active=True
+        )
+        self.org = self.create_organization(owner=self.owner)
+        self.first_team = self.create_team(organization=self.org)
+        self.proj1 = self.create_project(
+            name="proj1", organization=self.org, teams=[self.first_team]
+        )
+        self.proj2 = self.create_project(
+            name="proj2", organization=self.org, teams=[self.first_team]
+        )
+        self.superuser = self.create_user(
+            "superuser@example.com", is_superuser=True, is_staff=True, is_active=True
+        )
+        self.path = "sentry-api-0-internal-project-config"
+
+        self.p1_pk = self.create_project_key(self.proj1)
+        self.p2_pk = self.create_project_key(self.proj2)
+
+        projectconfig_cache.set_many(
+            {
+                self.p1_pk.public_key: "proj1 config",
+            }
+        )
+
+    def get_url(self, proj_id=None, key=None):
+        ret_val = reverse(self.path)
+        ret_val += "?"
```

Comment:
You can use `urllib.parse.urlencode` to construct the query string.

Reply:
@RaduW i think we want this:
```
from urllib.parse import urlencode, quote_plus

payload = {'username':'administrator', 'password':'xyz'}
result = urlencode(payload, quote_via=quote_plus)
# 'password=xyz&username=administrator'
```